api/lib/domain/events/handle-pix-plus-certifications-scoring.js

```diff
@@ -6,9 +6,46 @@ const { ReproducibilityRate } = require('../models/ReproducibilityRate');
 const AnswerCollectionForScoring = require('../models/AnswerCollectionForScoring');
 const { PIX_PLUS_DROIT } = require('../models/ComplementaryCertification');
 const { featureToggles } = require('../../config');
+const { PIX_DROIT_MAITRE_CERTIF, PIX_DROIT_EXPERT_CERTIF } = require('../models/Badge').keys;
 
 const eventTypes = [CertificationScoringCompleted, CertificationRescoringCompleted];
 
+async function _isAllowedToBeScored({
+  certifiableBadgeKey,
+  certificationCourseId,
+  complementaryCertificationCourseRepository,
+}) {
+  if (certifiableBadgeKey === PIX_DROIT_MAITRE_CERTIF || certifiableBadgeKey === PIX_DROIT_EXPERT_CERTIF) {
+    if (featureToggles.isComplementaryCertificationSubscriptionEnabled) {
+      return await complementaryCertificationCourseRepository.hasComplementaryCertification({
+        certificationCourseId,
+        complementaryCertificationName: PIX_PLUS_DROIT,
+      });
+    }
+  }
+  return true;
```

Comment:
suggestion: Je propose de mettre explicitement l'autorisation à pix edu (mais pas a tout). Est-ce que c'est pertinent?
```suggestion
if (certifiableBadgeKey === PIX_EDU || certifiableBadgeKey === PIX_EDU_XXX) {
  return true;
}

return false;
```